### PR TITLE
fix(docs-website): fix dark theme behaviour - I39

### LINKED ISSUE #39: Some elements are inconsistent with dark mode design formatting
# Bug Report 🐛
Some elements on the concerto docs homepage are rendered in light mode correctly but not in dark mode, sometimes causing difficulty in visibility and leading to inconsistent design.

## Expected Behavior
All elements on the page should change color according to the current display mode (light/dark) so as to maintain consistency and readability.

## Current Behavior
The elements labeled in the images below are observed to remain unchanged when the user switches to dark mode (or vice versa).
Screenshot (83)
Screenshot (84)

## Possible Solution
Fix the behaviour of the mentioned elements such that they change color according to the selected display mode.

## Steps to Reproduce

1. Go to the [concerto docs homepage](https://concerto.accordproject.org/).
2. Switch to dark mode.

### Desktop
 - OS: Windows 11
 - Browser: Microsoft Edge

### PR DESCRIPTION
**Pull request by apoorvsxna**
# Closes #39 
Fixed inconsistent behaviour of the elements mentioned in the [issue](https://github.com/accordproject/concerto-docs/issues/39), when switching the display theme (light to dark, or back).

### Changes
- Changed the styling of feature icons so that their colors adjust when switching themes.
- Included an image for the Accord Project logo in the navbar for use in the dark theme.

### Screenshots or Video
- Before-
Screenshot (83)
Screenshot (84)

- After-
Screenshot (85)
Screenshot (86)

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [x] Merging to `main` from `fork:branchname`

### CODE REVIEW

**Comment by apoorvsxna** [comment]
@mttrbrts @sanketshevkar Could you review this pull request and let me know if anything requires changing?

**Comment by sanketshevkar** [comment]
Thanks for your contributions! @apoorvsxna we'll have a look!

File: src/components/HomepageFeatures/styles.module.css [hunk 1]
```diff
@@ -10,4 +10,13 @@
...
+  stroke-width: 0.08px;
+  fill: #f5f6f7;
+  stroke: #f5f6f7;
 }
+
+/* Dark mode */
+[data-theme='light'] .featureSvg {
+  fill: #1C1E21;
+  stroke: #1C1E21; /* Change color in dark mode */
+}
```
Comment (DianaLease) [inline src/components/HomepageFeatures/styles.module.css:22]: Comments say dark mode but these values are for "light" mode, right? I think just the comments need updating.

**Review by DianaLease — CHANGES_REQUESTED** [review]
Code & preview LGTM. Just the inline comments need updating.

File: src/components/HomepageFeatures/styles.module.css (hunk 1 above)
Comment (apoorvsxna) [reply src/components/HomepageFeatures/styles.module.css]: Yep, seems like an oversight. Fixed now.

**Review by DianaLease — APPROVED** [review]
Looks good! Please [update your commits with DCO signoff](https://github.com/accordproject/concerto-docs/pull/40/checks?check_run_id=22869874115)

File: src/components/HomepageFeatures/styles.module.css (hunk 1 above)
Comment (apoorvsxna) [reply src/components/HomepageFeatures/styles.module.css]: Hi @DianaLease. I messed something up while trying to add the DCO signoff to my commit. I ended up created a [new PR](https://github.com/accordproject/concerto-docs/pull/41), complete with the suggested changes and the signoff. I hope it resolves the issue. Thanks for your patience.

**Comment by apoorvsxna** [comment]
> Looks good! Please [update your commits with DCO signoff](https://github.com/accordproject/concerto-docs/pull/40/checks?check_run_id=22869874115)

Hi @DianaLease. I messed something up while trying to add the DCO signoff to my commit. I ended up created a https://github.com/accordproject/concerto-docs/pull/41, complete with the suggested changes and the signoff. I hope it resolves the issue. Thanks!

**Pull request closed without merging**